javatests/arcs/core/storage/ReferenceModeStoreTest.kt

```diff
@@ -700,6 +700,32 @@ class ReferenceModeStoreTest {
     assertThat(store.backingStore.stores.size).isEqualTo(0)
   }
 
+  @Test
+  fun clearsbacking_andcontainer_stores() = runBlockingTest {
+    val store = ActiveStore<RefModeStoreData, RefModeStoreOp, RefModeStoreOutput>(
+      StoreOptions(
+        testKey,
+        CollectionType(EntityType(schema))
+      ),
+      this,
+      driverFactory,
+      ::testWriteBackProvider,
+      null
+    ) as ReferenceModeStore
+
+    assertThat(store.containerStore.closed).isEqualTo(false)
+    store.backingStore.stores.forEach { _, storeRecord ->
```

Comment:
This loop will be a no-op if `stores` is empty. So, although it looks like it's testing the right thing, it mightn't test anything. So it's probably a good idea to check it's not empty first. How many values do we expect to see here?

Additionally, it might be slightly cleaner to use `stores.values().forEach {` if you just want to iterate over the values.